include/grl/kuka/KukaJAVAdriver.hpp

```diff
@@ -288,9 +289,27 @@ namespace grl { namespace robot { namespace arm {
 
           auto name = fbbP->CreateString(std::get<RobotName>(params_));
 
-          auto kukaiiwaArmConfiguration = flatbuffer::CreateKUKAiiwaArmConfiguration(*fbbP,name,commandInterface_,monitorInterface_);
+          auto clientCommandMode = grl::flatbuffer::EClientCommandMode_POSITION;
```

Comment:
I am not sure what the flatbuffers defaults to when not specifically initialized. Maybe I am missing something about flatbuffers. Does it initialize appropriate fields without proper ordering?